file_paths.md

```diff
@@ -59,6 +59,11 @@ The files annotated by an '*' indicate that they are created by package installa
         routes.yaml                       # :route_file
         ssl                               # :ssldir                      /etc/puppet/ssl
 
+    /etc/puppetlabs/pxp-agent *
+        modules *                         # stores configuration files for pxp-agent modules
```

Comment:
I don't have a strong opinion. 
We've discussed about naming 'modules' several times. Last time [here](https://tickets.puppetlabs.com/browse/CTH-359?focusedCommentId=218679&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-218679) :)